proto/detectors.proto

```diff
@@ -977,6 +977,17 @@ enum DetectorType {
   Ngrok = 966;
   Replicate = 967;
   Postgres = 968;
+  AzureActiveDirectoryApplicationSecret = 969;
+  AzureCacheForRedisAccessKey = 970;
+  AzureCosmosdbKeyIdentifiable = 971;
+  AzureDevopsPersonalAccessToken = 972;
+  AzureFunctionKey = 973;
+  AzureMlWebServiceClassicIdentifiableKey = 974;
```

Comment:
nit: Lets also do the same for `Ml` -> `ML`.
`AzureMlWebServiceClassicIdentifiableKey` -> `AzureMLWebServiceClassicIdentifiableKey`